portal/get_patient_info.php

```diff
@@ -22,6 +22,22 @@
 // regenerating the session id to avoid session fixation attacks
 session_regenerate_id(true);
 
+// Settings that will override globals.php
+$ignoreAuth = 1;
+
+// Authentication
+require_once('../interface/globals.php');
+require_once(dirname(__FILE__) . "/lib/appsql.class.php");
+require_once("$srcdir/user.inc");
+
+use OpenEMR\Common\Auth\AuthHash;
+use OpenEMR\Common\Csrf\CsrfUtils;
+
+// landing page definition -- where to go if something goes wrong
+if (isset($_GET['site'])) {
+    $landingpage = "index.php?site=" . urlencode($_GET['site']);
+}
+
```

Comment:
This makes me nervous. Was this to fix a bug? I'll look into this a bit closer.

Reply:
Yep, will prob revert the changes in this file since need to block these things rather than give access (via the $ignoreAuth = 1; and globals.php call)